clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -1003,9 +1026,9 @@ class GCEUtil {
                                           GoogleExecutorTraits executor) {
     String serverGroupName = serverGroup.name
     Metadata instanceMetadata = serverGroup?.launchConfig?.instanceTemplate?.properties?.metadata
-    Map metadataMap = buildMapFromMetadata(instanceMetadata)
-    def httpLoadBalancersInMetadata = metadataMap?.(GoogleServerGroup.View.GLOBAL_LOAD_BALANCER_NAMES)?.tokenize(",") ?: []
-    def networkLoadBalancersInMetadata = metadataMap?.(GoogleServerGroup.View.REGIONAL_LOAD_BALANCER_NAMES)?.tokenize(",") ?: []
+    Map<String, String> metadataMap = buildMapFromMetadata(instanceMetadata)
+    def httpLoadBalancersInMetadata = metadataMap?.get(GLOBAL_LOAD_BALANCER_NAMES)?.tokenize(",") ?: []
+    def networkLoadBalancersInMetadata = metadataMap?.get(REGIONAL_LOAD_BALANCER_NAMES)?.tokenize(",") ?: []
```

Comment:
This was weird. You can't do `map?.(CONSTANT)` if `CONSTANT` is defined in the same class. It will always return `null`. `map?.(SomeOtherClass.CONSTANT)` works. And if you try to debug this thing and put a breakpoint here and use the "Evaluate Expression" feature, it also works. Go figure. I thought I was going mad until @robfletcher could confirm this behaviour 😆

Reply:
Sounds Groovy.